### PR TITLE
Update implementations.md

### PR DESCRIPTION
**Pull request by productml-ai**
Adding Azure Monitor in alphabetical order

### CODE REVIEW

**Review by mtwo — APPROVED** [review]

File: implementations.md
Comment (SergeyKanzhelev) [inline implementations.md]: would you mind adding links to keep it consistent with the rest of the document?

File: implementations.md
Comment (SergeyKanzhelev) [inline implementations.md]: nit:

```suggestion
**Website:** [Azure Monitor](https://azure.microsoft.com/services/monitor/)
```

**Comment by SergeyKanzhelev** [comment]
Marked as non substantive for IPR from ash-nazg.

File: implementations.md
Comment (productml-ai) [reply implementations.md]: Yes. Waiting for confirmation on specification links before applying.

Comment (productml-ai) [reply implementations.md]: Updated with available links

Comment (SergeyKanzhelev) [reply implementations.md]: Do you want to wait for more links or PR is ready to be merged?

(we will need more approvals anyway, just asking to understand the status)

Comment (productml-ai) [reply implementations.md]: Yes. I am in discussion with devs and we will probably move to "All technologies supported by Azure Monitor..." and link back to docs.microsoft.com since that is our source of truth for documentation. I should be able to close out by tomorrow.

Comment (productml-ai) [reply implementations.md]: We decided that it is best to navigate the user to docs.microsoft.com

**Review by SergeyKanzhelev — APPROVED** [review]

**Review by dyladan — APPROVED** [review]

**Review by arminru — APPROVED** [review]

**Pull request merged**
